opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/_view_instrument_match.py

```diff
@@ -52,6 +53,39 @@ def __init__(
         self._attributes_previous_point: Dict[frozenset, _PointVarT] = {}
         self._lock = Lock()
         self._instrument_class_aggregation = instrument_class_aggregation
+        self._name = self._view._name or self._instrument.name
+        self._description = (
+            self._view._description or self._instrument.description
+        )
+        if not isinstance(self._view._aggregation, DefaultAggregation):
+            self._aggregation = self._view._aggregation._create_aggregation(
+                self._instrument
+            )
+        else:
+            self._aggregation = self._instrument_class_aggregation[
+                self._instrument.__class__
+            ]._create_aggregation(self._instrument)
+
+    def conflicts(self, other: "_ViewInstrumentMatch"):
+        # pylint: disable=protected-access
+
+        result = (
+            self._name == other._name
+            and self._instrument.unit == other._instrument.unit
+            # The aggregation class is being used here instead of data point
+            # type since they are functionally equivalent.
+            and self._aggregation.__class__ == other._aggregation.__class__
```

Comment:
now does this compare with `type(..) is type(..)`?

Reply:
Tried it, but we get a Flake8 error: `do not compare types, use 'isinstance()' [E721]` Because of inheritance, using `isinstance` is not the same as comparing types so its better to use `==`.